instaloader/instaloader.py

```diff
@@ -204,6 +206,33 @@ def __init__(self,
         self.resume_prefix = resume_prefix
         self.check_resume_bbd = check_resume_bbd
 
+        self.slide = slide or ""
+        self.slide_start = 0
+        self.slide_end = 0
+        if self.slide != "":
+            splitted = self.slide.split('-')
+            if len(splitted) == 1:
+                if splitted[0] == 'last':
+                    #download only last image of a sidecar
+                    self.slide_end = -1
+                else:
+                    if int(splitted[0]) > 0:
+                        self.slide_start = self.slide_end = int(splitted[0])
+                    else:
+                        raise InvalidArgumentException("--slide parameter must be greather than 0.")
+            elif len(splitted) == 2:
+                if splitted[1] == 'last':
+                    self.slide_start = int(splitted[0])
+                    self.slide_end = -1
+                elif 0 < int(splitted[0]) < int(splitted[1]):
+                    self.slide_start = int(splitted[0])
+                    self.slide_end = int(splitted[1])
+                else:
+                    raise InvalidArgumentException("Invalid data for --slide parameter.")
+            else:
+                raise InvalidArgumentException("Invalid data for --slide parameter.")
+        else:
+            self.slide_start = 1
```

Comment:
```suggestion
            self.slide_start = 1

```

(add missing empty line between two methods)